docs/source/troubleshooting/index.rst

```diff
@@ -1,8 +1,8 @@
 Troubleshooting
 ===============
 
-This section includes common problems you might encounter while using the platform,
-and how to troubleshoot and debug them.
+This section includes common problems you might encounter while using the Brocade Workflow Composer platform,
```

Comment:
I think we should use `|fullname|` here, rather than "Brocade Workflow Composer", so it will render properly for StackStorm vs BWC docs.